stix_shifter_modules/elastic_ecs/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -793,36 +783,6 @@
         "references": "process_parent"
       }
     ],
-    "pe": {
```

Comment:
so not sure why you removed "pe" field mapping here. this "pe" is under "process" field. there are "pe" field mapped in couple of other places as well.

Reply:
@mdazam1942 we moved the `pe` field from the process to the file object.  Each process has a file associated with it (through the `binary_ref` in STIX 2.0 and `image_ref` in STIX 2.1), and our assumption was that the `pe` structure (and the `code_signature` structure, as well) will be mapped to the file structure, and the relationship between the process and the `pe`, or `code_signature` attributes will be kept through the `binary_ref` or `image_ref` pointer.   For example, this process:
```
"process" : {
  "pid" : 5244,
  "pe" : {
    "company" : "Microsoft Corporation",
    "file_version" : "10.0.17763.2145 (WinBuild.160101.0800)",
    "description" : "DSREG commandline tool",
    "imphash" : "382c77bfa0eee2ba2ba8671d108ad9a3",
    "original_file_name" : "dsregcmd.exe",
    "product" : "Microsoft® Windows® Operating System"
  },
  "entity_id" : "{8dfc401c-f31c-6175-5715-000000001b00}",
  "executable" : "C:\\Windows\\System32\\dsregcmd.exe",
  "working_directory" : "C:\\Windows\\system32\\",
  "command_line" : "C:\\Windows\\System32\\dsregcmd.exe $(Arg0) $(Arg1) $(Arg2)",
  "args" : [
    "C:\\Windows\\System32\\dsregcmd.exe",
    "$(Arg0)",
    "$(Arg1)",
    "$(Arg2)"
  ],
  "hash" : {
    "md5" : "d6957aceda86de523af0157800aa3c73",
    "sha256" : "ba79462455b6e216d0e7cd6fe36bf0eff8a0d9dd06358d1c97b1014016256618"
  },
  "name" : "dsregcmd.exe"
}
```
will be mapped to these two STIX objects ( process and file):
```
"1": {
  "type": "process",
  "pid": 5244,
  "x_unique_id": "{8dfc401c-f31c-6175-5715-000000001b00}",
  "binary_ref": "15",
  "command_line": "C:\\Windows\\System32\\dsregcmd.exe $(Arg0) $(Arg1) $(Arg2)",
  "name": "dsregcmd.exe"
},
"15": {
  "type": "file",
  "x_pe": {
    "company": "Microsoft Corporation",
    "file_version": "10.0.17763.2145 (WinBuild.160101.0800)",
    "description": "DSREG commandline tool",
    "original_file_name": "dsregcmd.exe",
    "product": "Microsoft\u00ae Windows\u00ae Operating System"
  },
  "name": "dsregcmd.exe",
  "parent_directory_ref": "16",
  "hashes": {
    "MD5": "d6957aceda86de523af0157800aa3c73",
    "SHA-256": "ba79462455b6e216d0e7cd6fe36bf0eff8a0d9dd06358d1c97b1014016256618"
  }
}
```